lib/update/engine.go

```diff
@@ -126,13 +126,6 @@ func (r *Engine) Complete(fsmErr error) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	err = r.operator.ActivateSite(ops.ActivateSiteRequest{
```

Comment:
This and below  should not be necessary as CompleteOperation/FailOperation transition the cluster to specified state.